task/pylint/0.2/pylint.yaml

```diff
@@ -20,7 +20,7 @@ spec:
   params:
     - name: image
       description: The container image with pylint
-      default: docker.io/cytopia/pylint@sha256:6e5b49b6d54cbd845b93139eeb3f4f558b07bf6a87001ce254782463df1443d2
+      default: registry.gitlab.com/pipeline-components/pylint:edge
```

Comment:
should we tag the image with digest as well?
/approve
/test .*

Reply:
Yeah but same as before ,how could we keep this updated ? We end up with very old version of tools in catalog

Reply:
We need some *dependabot-like* automation here.. something that would watch images and submit a PR on new tag published.